steps.mk

```diff
@@ -33,6 +33,7 @@ CT_STEPS := lnp			\
 			legends		\
 			announcement\
 			appimage    \
```

Comment:
move 'dark' before 'appimage'
it will allow releasing your pack with the AppImage format

Reply:
Will do!